src/local-storage.js

```diff
@@ -0,0 +1,64 @@
+import { editTodo, removeTodo } from './edit-remove.js';
+import { myArray } from './index';// eslint-disable-line
```

Comment:
- [ ] Please kindly do not disable linters, fix linters errors helps programmers to write clean and clear codebase 😁 😉